EIPS/eip-1898.md

```diff
@@ -1,39 +1,42 @@
 ---
 eip: 1898
-title: Add `blockHash` to JSON-RPC methods which accept a default block parameter.
+title: Add `blockHash` to defaultBlock methods
+description: Add `blockHash` option to JSON-RPC methods that currently support defaultBlock parameter.
 author: Charles Cooper (@charles-cooper)
+discussions-to: https://ethereum-magicians.org/t/eip-1898-add-blockhash-option-to-json-rpc-methods-that-currently-support-defaultblock-parameter/11757
+status: Review
```

Comment:
```suggestion
status: Draft
```
Stagnant EIPs cannot be moved to Review. They must go through the entire EIP process starting from Draft.

Reply:
Got it. thanks!